ohapi/utils_fs.py

```diff
@@ -168,6 +175,186 @@ def load_metadata_csv(input_filepath):
+        metadata provided.
+    """
+    if project_member_id is not None:
+        if not project_member_id.isdigit() or len(project_member_id) != 8:
+            raise ValueError(
+                'Error: for project member id: ', project_member_id,
+                ' and filename: ', filename,
+                ' project member id must be of 8 digits from 0 to 9')
+    if 'description' not in file_metadata:
+        raise ValueError(
+            'Error: for project member id: ', project_member_id,
+            ' and filename: ', filename,
+            ' "description" is a required field of the metadata')
+
+    if not isinstance(file_metadata['description'], str):
+        raise ValueError(
+            'Error: for project member id: ', project_member_id,
+            ' and filename: ', filename,
+            ' "description" must be a string')
+
+    if 'tags' not in file_metadata:
+        raise ValueError(
+            'Error: for project member id: ', project_member_id,
+            ' and filename: ', filename,
+            ' "tags" is a required field of the metadata')
+
+    if not isinstance(file_metadata['tags'], list):
+        raise ValueError(
+            'Error: for project member id: ', project_member_id,
+            ' and filename: ', filename,
+            ' "tags" must be an array of strings')
+
+    if 'creation_date' in file_metadata:
+        if not validate_date(file_metadata['creation_date'], project_member_id,
+                             filename):
+            return False
+
+    if 'md5' in file_metadata:
+        if not re.match(r'[a-z0-9]{32}', file_metadata['md5'],
+                        flags=re.IGNORECASE):
+            raise ValueError(
+                'Error: for project member id: ', project_member_id,
+                ' and filename: ', filename,
+                ' Invalid MD5 specified')
+
+    return True
+
+
+def review_metadata_csv_single_user(filedir, metadata, csv_in, n_headers):
+    """
+    Check validity of metadata for single user.
+
+    :param filedir: This field is the filepath of the directory whose csv
+        has to be made.
+    :param metadata: This field is the metadata generated from the
+        load_metadata_csv function.
+    :param csv_in: This field returns a reader object which iterates over the
+        csv.
+    :param n_headers: This field is the number of headers in the csv.
+    """
+    try:
+        if not validate_metadata(filedir, metadata):
+            return False
+        for row in csv_in:
+            if len(row) != n_headers:
+                raise ValueError(
+                    'Error: for filename: ', row[0],
+                    ' Number of columns (', len(row), ') does not match' +
+                    ' header row length (', n_headers, ')')
+        for filename, file_metadata in metadata.items():
+            is_single_file_metadata_valid(file_metadata, None, filename)
+    except ValueError as e:
+        printError(e)
```

Comment:
This function isn't actually printing anything, it's just returning a string. Errors aren't rendered visible to the user.